### PR TITLE
Added description of game-metadata

### PR DESCRIPTION
**Pull request by kstigen**

### CODE REVIEW

File: content/metadata.md [hunk 1]
```diff
@@ -79,12 +81,35 @@ An example of how to identify the cover image is as follows:
...
 "c418ac9f-fa75-4a72-9f3a-8ceacee850f2", "A Street or a Zoo?", https://books.digitallibrary.io/epub/en/c418ac9f-fa75-4a72-9f3a-8ceacee850f2.epub, 1
 ```
+
+## Games
+The GDL can host information about games and how to install/play the games.
+### Metadata as CSV
+#### Required
+* __ID__: Unique ID. Preferably a Google Play Store-identifier or an UUID.
+* __Title__: Title of the game
+* __Description__: A description of the game and what it is about
```
Comment (christer-io) [inline content/metadata.md]: Foreslår følgende: "A description of the game, including topic the games learning objectives "

File: content/metadata.md [hunk 2]
```diff
@@ -79,12 +81,35 @@ An example of how to identify the cover image is as follows:
...
-Example of a metadata csv-file:
+#### Example:
 ```csv
 Title, Link to epub, Reading Level
 "676719b9-f5bb-4ce7-83ca-e221cbc53c67", "Bath time for Chunnu and Munnu", https://books.digitallibrary.io/epub/en/676719b9-f5bb-4ce7-83ca-e221cbc53c67.epub, 1
 "c418ac9f-fa75-4a72-9f3a-8ceacee850f2", "A Street or a Zoo?", https://books.digitallibrary.io/epub/en/c418ac9f-fa75-4a72-9f3a-8ceacee850f2.epub, 1
 ```
+
+## Games
+The GDL can host information about games and how to install/play the games.
```
Comment (christer-io) [inline content/metadata.md]: Foreslår følgende: The GDL platform includes a separate category for games in selected languages, including information about the games and how to install/play them.

**Review by christer-io — COMMENTED** [review]
Har foreslått to små endringer i teksten.

File: content/metadata.md (hunk 1 above)
Comment (kstigen) [reply content/metadata.md]: Jeg forstår ikke helt setningen din der. Er det følgende du mener: `A description including topic and the learning objectives of the game.`

**Review by kievu — APPROVED** [review]

**Pull request merged**